src/blog/2019-04-30-hub-enhancements-apr-2019/index.md

```diff
@@ -0,0 +1,44 @@
+
+![Before after Projects List](before-after.gif)
+
+You will have noticed other improvements to the consitency of buttons and general UX. There are more changes to come as we redesign some of the interface to support upcoming features, and we will announce them as they are released. But you will also see consistent incremental improvements to the UI until then.
+
+## Session Queueing
+
+When launching a Stencila Desktop session through Stencila Hub, there is a chance that an execution host is not ready and has to be started. This might take a couple of minutes – although we have work in the pipeline to reduce startup time in the future. While session queueing is not new to Stencila Hub, we have made a number of fixes and improvements to improve reliability.
+
+![Waiting for session](waiting-spinner.png)
+
+
+The above spinner is probably something you have seen before when launching Stencila Desktop. It might take a minute or two to get a session but with the improvements we have done there shouldn't be any timeouts or errors – it will launch eventually.
+
+## Deployment
+
+As well as the improvements to the deployment process (explained in the [New Deployment Process blog post](/blog/2019-04-17-consistent-versioning-deployment-django-express/)) there have also been a few Hub specific changes. The load balancer uses the `system-status` endpoint to check if a deployment instance is ready to receive connections. It now returns an error if there are unapplied Django migrations. This means that connections will continue to be served by an existing instance until the database is in sync with the application code. Once the migration occurs requests will automatically be sent to the new instance.
+
+We've also updated the Kubernetes deployment script to use rolling updates. We can now roll out updates and because the load balancer and Kubernetes checks the `system-status` endpoint, it waits until the new deployment is ready before switching over. Once it's ready the switch is instantaneous so we can deploy Hub (or Stencila Cloud) with no downtime.
+
+## Internet Explorer Deprecation
+
+Due to the use of some advanced JavaScript features, Stencila Hub has never really supported Internet Explorer. In the past, there was no notification of this to Internet Explorer users. Now, Internet Explorer users get their own special screen to let them know they are on an unsupported browser, instead of things just failing for no reason.
+
+Our roadmap includes a fully enumerated list of supported browsers and we may later down-compile ES2015+ to JavaScript using Babel (or similar) to add Internet Explorer support, but there are no concrete plans surrounding that yet.
+
+## Conclusion
+
+Thank you for reading about our latest changes to Stencila Hub. We have more updates in the works focusing on more UI enhancements and integration with more third party services. See you next time!
```

Comment:
Not essential but the conclusion could be a good place to potentially plug the the upcoming _Catching Up With Stencila call_ on May 6th.
I think [this link](https://calendar.google.com/event?action=TEMPLATE&tmeid=ZDcyYXJ1aDFhOWk3NXVsamZrczc0ZmVrcWdfMjAxOTA1MDZUMjAwMDAwWiBzdGVuY2kubGFfYms3NGZnYTAzM24zMjVkczVtbnE3bWI2b2tAZw&tmsrc=stenci.la_bk74fga033n325ds5mnq7mb6ok%40group.calendar.google.com&scp=ALL) works but only if signed into Google